assignments/closure.js

```diff
@@ -1,10 +1,17 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
-
+const name = "Jordan";
+function speak (){
+  console.log ('I really liked michael ${name}');
+}
+speak()
```

Comment:
look into closures a little more. their strength is in the 2 pass compiler of JS and the way they store values for later use.